.env.template

```
@@ -21,3 +21,5 @@ STOMP_LOGIN: guest
 STOMP_PASS: guest
 STOMP_USER_ACTIONS_TOPIC: /topic/user_actions
 STOMP_CLIENT_NAME: dev-client
+
+PL_MARKETPLACE_USER_ROLES_TOKEN: token
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The PL_MARKETPLACE_USER_ROLES_TOKEN: token key has incorrect delimiter

(IncorrectDelimiter)